### ISSUE TITLE
Renyi entropy

### ISSUE THREAD

**Issue by NicholasSynesi** [LucaYoy/Many-quantum-bodies-using-quantum-computing#45]
Could be interesting to look at something other than the von Neumann entropy, and shouldn't be too hard to implement.

**Comment by NicholasSynesi** [comment]
@as2457 

As a quick aside I tried plotting the Renyi entropy as it was a very simple change. I found a few resources already and in the write up I plan to just make it a quick aside, but I was wondering if these graphs look like they should? 

Here is my alpha = 0

![image](https://user-images.githubusercontent.com/117459444/229097837-ba989405-8baf-4d1f-8703-b7eb7f3bc27a.png)

which gives an interesting graph. You mentioned this corresponds to the purity in the notes, is this something that can be concluded from this graph?
Also, is there any other plots of the Renyi entropy that you think could be interesting?
